dockertest/output/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+API compatibility: on 2016-04-29 output.py got split up into a
+package with four smaller, more maintainable submodules. This init
+allows existing clients to continue importing as a single unit.
+"""
+
+# ARGH. Without this, 'import xceptions' in a submodule fails because
```

Comment:
heh, ya, this stuff will drive you crazy.  it's all about namespaces, forget about the directory structures.  Think namespaces, what names need to be anchored where, and then make the directory structure and ``__init__``'s to realize it.